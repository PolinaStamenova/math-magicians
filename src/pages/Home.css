.home-holder {
  padding: 0 5rem;
  margin-top: 3rem;
}

.home-subtitle {
  margin-top: 5rem;
}

.home-third-title {
  margin-top: 2rem;
}
