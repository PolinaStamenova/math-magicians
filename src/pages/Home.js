/* eslint-disable react/prefer-stateless-function, quotes */

import React from "react";
import "./Home.css";

export default function Home() {
  return (
    <div className="home-holder">
      <h2>This is the HOME page</h2>
      <p className="home-subtitle">
        Note: Behind the scenes a renders anwith a real href, so people using
        the keyboard for navigation or screen readers will still be able to use
        this app.
      </p>
      <p className="home-third-title">
        Note: Behind the scenes a renders anwith a real href, so people using
        the keyboard for navigation or screen readers will still be able to use
        this app.
      </p>
    </div>
  );
}
