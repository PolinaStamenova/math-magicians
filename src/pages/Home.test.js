import React from "react";
import renderer from "react-test-renderer";
import Home from "./Home";

it("expect home page to match working snapshot", () => {
  const homePage = renderer.create(<Home />).toJSON();
  expect(homePage).toMatchSnapshot();
});
