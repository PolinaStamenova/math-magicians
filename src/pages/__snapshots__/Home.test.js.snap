// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect home page to match working snapshot 1`] = `
<div
  className="home-holder"
>
  <h2
    className="home-title"
  >
    Welcome to my page!
  </h2>
  <p
    className="home-text"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi.
  </p>
  <p
    className="home-second-text"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi.Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit.
  </p>
</div>
`;
